src/libraries/Microsoft.Extensions.Logging.Console/src/LoggerColorBehavior.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    /// <summary>
+    /// Determines when to use color when logging messages.
+    /// </summary>
+    public enum LoggerColorBehavior
+    {
+        /// <summary>
+        /// Use the default color behavior
+        /// </summary>
+        /// <remarks>
+        /// Enables color except when the console output is redirected.
```

Comment:
>Enables color except when the console output is redirected. [](start = 12, length = 59)

I think moving this to the summary will be better I guess.

Reply:
https://github.com/dotnet/runtime/pull/40221#discussion_r464003052

Reply:
Personally I am seeing it is too much jump to reach to the simple info. having it in the summary is not too much description. remarkas usually describe more details if needed like if redirecting to some stream which will get displayed later again or something like that to tell more information what is the expected behavior. 

anyway, you don't have to block on that though.